tests/samplers_tests/test_samplers.py

```diff
@@ -43,8 +43,7 @@ def test_pickle_random_sampler(seed):
 
     sampler = optuna.samplers.RandomSampler(seed)
     restored_sampler = pickle.loads(pickle.dumps(sampler))
-    assert sampler._rng != restored_sampler._rng
```

Comment:
Removed this assertion because it seems meaningless. The equality check of two RNGs seems always to result in `False` (see below):
```python
assert numpy.random.RandomState(10) != numpy.random.RandomState(10)

rng =  assert numpy.random.RandomState(10)
assert rng != copy.deepcopy(rng)
```